### PR TITLE
i have changed some translations

### PR DESCRIPTION
**Pull request by Isabirye1515**
file:///home/elai/Pictures/Screenshots/Screenshot%20from%202024-04-25%2021-48-32.png
file:///home/elai/Pictures/Screenshots/Screenshot%20from%202024-04-25%2021-49-06.png
file:///home/elai/Videos/Screencasts/Screencast%20from%2025-04-24%2021:42:23.webm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# OpenELIS Global 2
+sudo # OpenELIS Global 2
```
Comment (sherrif10) [inline README.md:1]: is this change necessary ?

File: dev.docker-compose.yml [hunk 2]
```diff
@@ -140,4 +140,4 @@ volumes:
   db-data:
   key_trust-store-volume:
   certs-vol:
-  keys-vol:
\ No newline at end of file
+  keys-vol:
```
Comment (sherrif10) [inline dev.docker-compose.yml:143]: Is this change neccessary ?

File: docker-compose.yml~ [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        image: itechuw/certgen:main
+        restart: always
+        environment:
+            - KEYSTORE_PW="kspass"
+            - TRUSTSTORE_PW="tspass"
+        networks:
+            - default
+        volumes:
+            -  key_trust-store-volume:/etc/openelis-global
+            -  keys-vol:/etc/ssl/private/
```
Comment (sherrif10) [inline docker-compose.yml~:14]: Clean up this PR and only include what is required. i would advise to only focus on tickets  created by @mozzy11  to get more hands on skills.

Comment (Isabirye1515) [reply docker-compose.yml~:14]: Am not understanding its my first time to push but every thing worked on my machine here is the tickethttps://github.com/I-TECH-UW/OpenELIS-Global-2/issues/708

Comment (Isabirye1515) [reply docker-compose.yml~:14]: Screenshot from 2024-04-25 21-49-06
[Screencast from 25-04-24 21:42:23.webm](https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/150329678/f00a9f62-9cc0-4c38-9af3-fd9b3924b66f)

**Comment by Isabirye1515** [comment]
[Screencast from 25-04-24 21:42:23.webm](https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/150329678/8b9a4c89-b849-4e30-801f-83a4d66d3649)

**Comment by Isabirye1515** [comment]
ita my first time to push on github please help me how can i clean up my oull request

**Comment by Isabirye1515** [comment]
i worked on this and it was merged
